src/ui/screens/settings/ViewPreferences.qml

```diff
@@ -31,6 +31,16 @@ MZViewBase {
 
         spacing: MZTheme.theme.windowMargin
 
+        MZLinkRow {
+            objectName: "androidStartAtBootLink"
+
+            accessibleName: _startAtBootTitle
+            title: _startAtBootTitle
+            subLabelText: MZI18n.SettingsStartAtBootSubtitle
+            visible: Qt.platform.os === "android"
+            destinationUrl: "https://support.mozilla.org/kb/how-enable-always-vpn-android"
```

Comment:
Suggestion: For the sake of consistency with `MZExternalLinkListItem{}` and `MZToggleRow` we could
- move this sumo URL to constants.h and then register a new url label for it in mozillavpn.cpp
- change the onClicked handler on L73 of `MZLinkRow` to be `linkRow.clicked()` and add `signal clicked()` somewhere around L20 of `MZLinkRow` (check out MZToggleRow as an example)
- drop the `destinationUrl` property from `MZLinkRow` and then change this line to `onClicked: MZUrlOpener.openUrlLabel("theNewLabel")`

Reply:
Thanks, updated